app/src/main/java/com/example/family_bank_app/TransactionEntity.java

```diff
@@ -3,17 +3,21 @@
 import androidx.annotation.NonNull;
 import androidx.room.ColumnInfo;
 import androidx.room.Entity;
+import androidx.room.ForeignKey;
 import androidx.room.PrimaryKey;
 
 // Store the transactions related to each account, one to many with account Entity
 //https://developer.android.com/training/data-storage/room/relationships
-@Entity
+@Entity(foreignKeys = @ForeignKey(entity = AccountEntity.class,
+        parentColumns = "accountUid",
+        childColumns = "transactionUid",
+        onDelete = ForeignKey.CASCADE))
```

Comment:
Are you using this? The same code is already created in the AccountWithTransactions activity.